src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -123,6 +179,64 @@ export class MenuButtonsMetaInfo extends React.PureComponent<Props> {
                 </div>
               ) : null}
             </div>
+            {calculatedStats.size > 0 ? (
+              <details>
+                <summary className="arrowPanelSubTitle">
+                  Profiler Overhead
+                </summary>
+                <div className="arrowPanelSection">
+                  <div className="metaInfoGrid">
+                    <div />
+                    <div>Mean</div>
+                    <div>Max</div>
+                    <div>Min</div>
+                    {Array.from(calculatedStats).map(([key, val]) => [
+                      <div key={key}>{key}</div>,
+                      <div key={key + 'mean'}>
+                        {formatNanoseconds(val.mean)}
+                      </div>,
+                      <div key={key + 'max'}>{formatNanoseconds(val.max)}</div>,
+                      <div key={key + 'min'}>{formatNanoseconds(val.min)}</div>,
+                    ])}
+                  </div>
+
+                  {overheadDurations !== 0 ? (
+                    <div className="metaInfoRow">
+                      <span className="metaInfoWideLabel">
+                        Overhead Durations:
+                      </span>
+                      <span className="metaInfoValueRight">
+                        {formatNanoseconds(
+                          overheadDurations / totalSamplingCount
+                        )}
+                      </span>
+                    </div>
+                  ) : null}
+                  {overheadPercentage !== 0 ? (
+                    <div className="metaInfoRow">
+                      <span className="metaInfoWideLabel">
+                        Overhead Percentage:
+                      </span>
+                      <span className="metaInfoValueRight">
+                        {formatPercent(overheadPercentage / totalSamplingCount)}
+                      </span>
+                    </div>
+                  ) : null}
+                  {profiledDuration !== 0 ? (
+                    <div className="metaInfoRow">
+                      <span className="metaInfoWideLabel">
+                        Profiled Duration:
+                      </span>
+                      <span className="metaInfoValueRight">
+                        {formatNanoseconds(
+                          profiledDuration / totalSamplingCount
+                        )}
+                      </span>
+                    </div>
+                  ) : null}
+                </div>
+              </details>
+            ) : null}
```

Comment:
I think this is self-contained enough to move to a separate component, along with the computations above.